tools/CoqMakefile.in

```diff
@@ -718,6 +725,10 @@ endif
 $(VOFILES): %.vo: %.v
 	$(SHOW)COQC $<
 	$(HIDE)$(TIMER) $(COQC) $(COQDEBUG) $(TIMING_ARG) $(COQFLAGS) $(COQLIBS) $< $(TIMING_EXTRA)
+ifeq '$(COQMF_COQ_NATIVE_COMPILER_DEFAULT)' "yes"
```

Comment:
@ppedrot can't you replace this test with a test on the .vo ? (implicit or not in the call to COQNATIVE)

Reply:
What should be tested exactly? vo files do not contain anything native-related.

Reply:
(This is the whole point btw: vo files should be native-agnostic if we ever want to have a split package infrastructure.)

Reply:
But you said that calling COQNATIVE on a vo compiled with -native off raises an error. How is it.

Reply:
That's not exactly what I said. If a vo file has dependencies whose native files are missing, then the native compilation process will fail. This is unrelated to coqnative per se, coqc behaves exactly the same.

As I argued in the CEP, `-native-compiler no` means two things at once that do not need to be correlated in general, namely 1. use `native_compute` and 2. generate cmxs files when compiling vo. coqnative is insensitive to the configure flag insofar as if it has accesses to the vo, it doesn't care about the origin of the cmxs files (either from coqc or coqnative). Furthermore, it could very well generate recursively the missing dependencies from the vo files.